distribution/docker/druid.sh

```diff
@@ -40,6 +40,9 @@
 # - DRUID_CONFIG_COMMON -- full path to a file for druid 'common' properties
 # - DRUID_CONFIG_${service} -- full path to a file for druid 'service' properties
 # - DRUID_SINGLE_NODE_CONF -- config to use at runtime. Choose from: {large, medium, micro-quickstart, nano-quickstart, small, xlarge}
+# - DRUID_ADDITIONAL_CONF_DIRS -- full path to a directory containing configuration files needed for extentions and will be copied to the common directory.
```

Comment:
let's just rename this to DRUID_ADDITIONAL_CLASSPATH. Also, we are not really copying anything. Please add a note that users should ensure that the paths references in `DRUID_ADDITIONAL_CLASSPATH` get mounted on containers.

Reply:
Thanks for the quick response :). Changed it :)